### ISSUE TITLE
Transparency not respected in KDE Plasma 6

### ISSUE THREAD

**Issue by AlekEagle** [HimDek/Utterly-Sweet-Plasma#3]

**Comment by muneebusmani** [comment]
does this problem still exists ? or is it fixed now

**Comment by AlekEagle** [comment]
I'm still having this problem with this when it comes to non-fullscreen application launchers, the system tray, and the primary desktop panel.

**Comment by muneebusmani** [comment]
> I'm still having this problem with this when it comes to non-fullscreen application launchers, the system tray, and the primary desktop panel.

Could you elaborate the issue with screenshots?

**Comment by AlekEagle** [comment]
Yes I can, the menus I have open here should have transparency (at least according to the example screenshots on the readme) ![examples](https://cdn.alekeagle.me/IINJUMtCtL.png)

**Comment by muneebusmani** [comment]
> Yes I can, the menus I have open here should have transparency (at least according to the example screenshots on the readme) ![examples](https://cdn.alekeagle.me/IINJUMtCtL.png)

Have you set the transparency via settings? Are you on Wayland or x11? Also tell me which options did you modify in order to apply this theme, and did you have transparency effects before using this theme ?

**Comment by AlekEagle** [comment]
Yes I have enabled transparency via settings. I am on Wayland (though I also don't have transparency on my other computer running X11). I simply applied the theme via global themes. yes I had transparency effects via other themes before using this theme. I also have a built from source version of Kvantum to enable transparency for KDE 6.

**Comment by HimDek** [comment]
Please keep updating the system, the Global themes and its dependencies and see it it works.

**Comment by pneumus** [comment]
Did you enable "blur" in effects?
